app/scripts/services/metrics.js

```diff
@@ -169,6 +175,44 @@ angular.module("openshiftConsole")
       });
     };
 
+    // Returns custom metrics available for a particular pod
+    var getCustomMetrics = function(pod) {
+      var namespace = pod.metadata.namespace;
+      var podName = pod.metadata.name;
+
+      return getMetricsURL().then(function(metricsURL) {
+
+        if (!metricsURL) {
+          return null;
+        }
+
+        var url = metricsURL + "/metrics";
+
+        var params = {
+          tags: "custom_metric:true,pod_name:" + podName
```

Comment:
Missed this on first review, but should we use pod UID instead in case a pod with the same name is created twice? (For pods that don't have generated names.)